adb_graphics/figures/maps.py

```diff
@@ -310,8 +316,9 @@ def draw(self, show=False): # pylint: disable=too-many-locals, too-many-branches
             self._wind_barbs(add_wind)
 
         # Add field values at airports
+        add_airports = self.field.vspec.get('plot_airports', True)
         annotate = self.field.vspec.get('annotate', False)
-        if annotate and self.map.model not in ['global']: # airports are too dense in global
+        if add_airports and annotate and 'global' not in self.map.model: # too dense in global
```

Comment:
Good point.  I had that in case we wanted to plot airports for Alaskan (or other) subdomains for global.  If that's wanted at some point, the logic here will need to be changed.